src/components/Button/Button.scss

```diff
@@ -0,0 +1,751 @@
+@import "../../scss/global.scss";
+
+.btn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  &.default {
+    min-width: 9.4rem;
+    min-height: 3.5rem;
+
+    span {
+      font-size: 1.2rem;
+    }
+  }
+
+  &.tiny {
+    min-width: 4.4rem;
+    min-height: 2.4rem;
+
+    span {
+      font-size: 1rem;
+    }
+  }
+
+  &.big {
+    min-width: 5.4rem;
+    min-height: 4.5rem;
+
+    span {
+      font-size: 1.3rem;
+    }
+  }
+
+  &.primary {
+    &.filled {
+      background-color: $main-theme;
+      border-radius: 0.25rem;
+      border: solid 0.2rem $main-theme;
+      color: #ffffff;
+
+      &.inverse {
+        background-color: $main-theme-lighter;
+        border-color: $main-theme-lighter;
+        color: $gray-dark;
+
+        &:focus,
+        &.focus {
+          background-color: $main-theme-lighter;
+        }
+
+        &:hover,
+        &.hover {
+          background-color: #dff4ff;
```

Comment:
I'm not sure why, but in Storybook, the text color is changing in the inverted fill hover state (it shouldn't change). I can't seem to pin down where this is in the scss though:
<img width="459" alt="Screenshot 2023-08-08 at 12 08 41 PM" src="https://github.com/CDCgov/cdc-react/assets/70541324/29a6e580-666d-446c-b28e-544a182edd67">

Reply:
Updated